README.md

```diff
@@ -128,3 +131,5 @@ See also <http://repo1.maven.org/maven2/org/junit/> for releases and <https://os
 [JUnit 5 User Guide]: http://junit.org/junit5/docs/current/user-guide/
 [Prototype]: https://github.com/junit-team/junit5/wiki/Prototype
 [Twitter]: https://twitter.com/junitlambda
+[StackOverflow]: https://stackoverflow.com/questions/tagged/junit5
+[Gitter]: https://gitter.im/junit-team/junit5
```

Comment:
For consistency, these new links should be inserted alphabetically into this list.